Libraries/Opc.Ua.Server/Subscription/IMonitoredItem.cs

```diff
@@ -123,15 +123,26 @@ public interface ITriggeredMonitoredItem
         /// <summary>
         /// The identifier for the item that is unique within the server.
         /// </summary>
-        uint Id { get; } 
-        
+        uint Id { get; }
+
         /// <summary>
         /// Flags the monitored item as triggered.
         /// </summary>
         /// <returns>True if there is something to publish.</returns>
         bool SetTriggered();
     }
 
+    /// <summary>
+    /// A monitored item were the base subscription can be transfered
+    /// </summary>
+    public interface ITransferableMonitoredItem
+    {
+        /// <summary>
+		/// The session that owns the monitored item.
```

Comment:
>		 [](http://example.com/codeflow?start=0&length=2)

no tabs please

Reply:
Changed to spaces (but we all know that tabs are better 😎)